repos/fdbt-site/src/pages/manageProductGroup.tsx

```diff
@@ -0,0 +1,510 @@
+                                                    <em>
+                                                        You currently have no single or return products that can be
+                                                        added to product group.
+                                                    </em>
+                                                </p>
+                                            )}
+                                        </div>
+                                        <div
+                                            className="govuk-tabs__panel govuk-tabs__panel--hidden"
+                                            id="flat-fare-products"
+                                        >
+                                            {otherProducts.length > 0 ? (
+                                                <div>
+                                                    <h2 className="govuk-heading-m govuk-!-margin-bottom-6">
+                                                        Flat fare products
+                                                    </h2>
+
+                                                    {otherProducts.length > 0 ? (
+                                                        buildOtherProductSection(
+                                                            indexCounter + otherProducts.length,
+                                                            productsSelected,
+                                                            setProductsSelected,
+                                                            otherProducts,
+                                                        )
+                                                    ) : (
+                                                        <p className="govuk-body-m govuk-!-margin-top-5">
+                                                            <em>
+                                                                You currently have no flat fare products that can be
+                                                                added to product group.
+                                                            </em>
+                                                        </p>
+                                                    )}
+                                                </div>
+                                            ) : null}
+                                        </div>
+                                    </div>
+                                </div>
+                            )}
+
+                            <button
+                                type="submit"
+                                className={`govuk-button${
+                                    productsToDisplay.length === 0 ? ' govuk-visually-hidden' : ''
+                                }`}
+                                id="continue-button"
+                            >
+                                Create Product Group
```

Comment:
The text of this button needs to change if its in edit mode (Save product group)